integration-tests/src/test/resources/burningwave.static.properties

```diff
@@ -0,0 +1,4 @@
+managed-logger.repository=autodetect
```

Comment:
@Bouncheck , why do we need it ?

Reply:
I added this configuration to hide the banner and logs with config options that get printed to the console.